### PR TITLE
Update Joshua Wee PPP

### PR DESCRIPTION
**Pull request by jwee97**

### CODE REVIEW

**Comment by joshlay96** [comment]
WAIT, dont merge yet pls

**Comment by joshlay96** [comment]
image

Can change this line for me before merging? Psps

**Comment by Deunitato** [comment]
> image
> 
> Can change this line for me before merging? Psps

It is supposed to find activity by title and location

**Review by Deunitato — APPROVED** [review]

**Pull request merged**
